src/modules/keyboardmanager/common/Shortcut.h

```diff
@@ -169,10 +172,10 @@ class Shortcut
     int GetCommonModifiersCount(const Shortcut& input) const;
 
     // Function to check if the two shortcuts are equal or cover the same set of keys. Return value depends on type of overlap
-    static KeyboardManagerHelper::ErrorType DoKeysOverlap(const Shortcut& first, const Shortcut& second);
+    static Helpers::ErrorType DoKeysOverlap(const Shortcut& first, const Shortcut& second);
 
     // Function to check if the shortcut is illegal (i.e. Win+L or Ctrl+Alt+Del)
-    KeyboardManagerHelper::ErrorType IsShortcutIllegal() const;
+    Helpers::ErrorType IsShortcutIllegal() const;

```

Comment:
This should also be changed to be a static method that receives a `Shortcut` as input argument.
The reason is that this class should not reference `ErrorType` since this is an enumeration specific for the Editor, and right now we are forcing the Engine to depend on stuff that is reserved for the Editor.
And it should be moved to `BufferValidationHelpers.cpp`